package.json

```diff
@@ -29,11 +29,13 @@
     "gulp-autoprefixer": "^3.1.0",
     "gulp-eslint": "^1.0.0",
     "gulp-if-else": "^1.0.3",
+    "gulp-inject-string": "^1.1.0",
     "gulp-lintspaces": "^0.3.0",
     "gulp-minify-css": "^1.1.6",
     "gulp-mustache": "^2.2.0",
     "gulp-rename": "^1.2.2",
-    "gulp-sass": "^2.0.1",
+    "gulp-sass": "~2.2.0",
```

Comment:
Checked and it looks fine. @roland-vachter can you rebase this with the current master then we should be ok to merge.